jsonValueEval.go

```diff
@@ -0,0 +1,68 @@
+package gojsonsm
+
```

Comment:
This is used for testing alongside gojsonsm. We can remove it later once it's done.